src/cljs/rems/spa.cljs

```diff
@@ -280,11 +280,10 @@
 (defn footer []
   [:footer.footer
    [:div.container
-    [:div.navbar
-     [:div.navbar-text (text :t/footer)]
-     (when (config/dev-environment?)
-       [:div.dev-only
-        [dev-reload-button]])]]])
+    (when (config/dev-environment?)
+      [:div.dev-reload-button.dev-only
+       [dev-reload-button]])
```

Comment:
I think it doesn't make sense to wrap `dev-reload-button` in a use site to `.dev-reload-button`, it should be instead rolled into the button component.

Reply:
Do you think I should keep the `dev-only` class? It doesn't seem to have a use right now.

Reply:
I removed it too.